packages/ember-data/tests/unit/model/errors_test.js

```diff
@@ -30,7 +30,6 @@ function unexpectedSend(eventName) {
 }
 
 test("add error", function() {
-  expect(6);
```

Comment:
That is true, but I am more worried about people using tests as an example and then not adding expects, and then believing their tests pass by mistake. Or someone refactoring the test to be async. Not sure it's a reasonable fear though